storage/xml-api/serviceaccount-appengine-sample/pom.xml

```diff
@@ -17,10 +31,15 @@
   <packaging>war</packaging>
 
   <properties>
+    <maven.compiler.target>1.6</maven.compiler.target>
+    <maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
/shruggie That's what it was. I was a bit surprised, too. I'll update that one, too.